editoast/src/main.rs

```diff
@@ -680,17 +714,26 @@ mod tests {
     #[rstest]
     async fn import_railjson_ko_file_not_found(db_pool: Data<DbPool>) {
         // GIVEN
+        let railjson_path = "non/existing/railjson/file/location";
         let args: ImportRailjsonArgs = ImportRailjsonArgs {
             infra_name: "test".into(),
-            railjson_path: "non/existing/railjson/file/location".into(),
+            railjson_path: railjson_path.into(),
             generate: false,
         };
 
         // WHEN
-        let result = import_railjson(args, db_pool).await;
+        let result = import_railjson(args.clone(), db_pool).await;
 
         // THEN
-        assert!(result.is_err())
+        assert!(result.is_err());
+        let error = CliError::new(
+            1,
+            format!(
+                "❌ Railjson file not found, Path: {}",
+                args.railjson_path.to_string_lossy()
+            ),
+        );
+        assert_eq!(result.unwrap_err().downcast_ref::<CliError>(), Some(&error));
```

Comment:
Maybe we should only check the error code to avoid the test breaking whenever we reword the error.